### PR TITLE
[ALLUXIO-1776] Add unit tests for IdUtils

### PR DESCRIPTION
**Pull request by breakdawn**
Add two unit tests for IdUtils.
mvn test passed.

### CODE REVIEW

**Comment by apc999** [comment]
ok to test

**Comment by apc999** [comment]
@breakdawn now the PR is good as your commit is linked to your account!

File: core/common/src/test/java/alluxio/util/IdUtilsTest.java [hunk 1]
```diff
@@ -30,4 +30,27 @@ public void getRandomNonNegativeLongTest() throws Exception {
...
   }
+
+  /**
+   * Tests if output of {@link IdUtils#createFileId(long)} is valid.
+   */
+  @Test
+  public void createFileIdTest() throws Exception {
+    long containerId = 1;
+    long fileId = IdUtils.createFileId(containerId);
+    Assert.assertTrue(fileId != -1);
```
Comment (apc999) [inline core/common/src/test/java/alluxio/util/IdUtilsTest.java]: could you make it like this to be consistent with current Alluxio junit test style?
```java
Assert.assertNotEqual(-1, fileId)
```

File: core/common/src/test/java/alluxio/util/IdUtilsTest.java [hunk 2]
```diff
@@ -30,4 +30,27 @@ public void getRandomNonNegativeLongTest() throws Exception {
...
+   * Tests if output of {@link IdUtils#createRpcId()} is non-empty.
+   * Also tests for randomness property.
+   */
+  @Test
+  public void createRpcIdTest() throws Exception {
+    String first = IdUtils.createRpcId();
+    Assert.assertTrue(first != null && !first.isEmpty());
+    String second = IdUtils.createRpcId();
+    Assert.assertTrue(second != null && !second.isEmpty());
+    Assert.assertTrue(!first.equals(second));
```
Comment (apc999) [inline core/common/src/test/java/alluxio/util/IdUtilsTest.java]: also, `assertNotEqual` for line 54?

**Comment by apc999** [comment]
@breakdawn I left two minor comments. PTAL. Thanks~

**Comment by breakdawn** [comment]
Fixed according to the comments.

**Comment by apc999** [comment]
+1 LGTM. thanks for the patch!

**Comment by breakdawn** [comment]
Thanks for your help. Alluxio is a great project.

**Pull request merged**